pkg/apis/agones/v1/apihooksfake.go

```diff
@@ -20,42 +20,43 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// FakeAPIHooks is a stubabble, fake implementation of APIHooks
-type FakeAPIHooks struct {
+// fakeAPIHooks is a stubabble, fake implementation of APIHooks
+// This needs to be private, so it doesn't get picked up by the DeepCopy() generation toolkit.
+type fakeAPIHooks struct {
```

Comment:
This struct is now duplicated here and in `pkg/testing`. I don't think there are any dependencies in `pkg/testing` that prevent it from being included into this package, so can we just move this file there (and remove it from here) so that it isn't duplicated?

Reply:
I also figured (possibly incorrectly?) that stubbing was only really necessary for `pkg/apis/agones/v1` - so a generic no-op version that was shareable seemed to make the most sense.

Reply:
Thanks. @zmerlynn - can you take a look when you get a chance to see if there is a better way to do this without code duplication? Until then, merging this to get the make target fixed seems appropriate.